todo.js

```diff
@@ -15,3 +15,39 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+const fs = require('fs');
+let todoList = fs.readFileSync("todos.txt","utf-8");
+let command = process.argv[2];
+let readLineSync = require("readLine-sync");
+let TODO = {};
+function list() {
```

Comment:
Don't use global variables. Write simple functions that accepts whatever arguments they need to do their work.

I'd expect something like `printTodoList(todoList)`, `addTask(todoList, task)`